### DISCUSSION TITLE
Auth: missing migrations & configuration

### DISCUSSION THREAD

**Discussion by christopherfriedrich** [adonisjs/core#1319] [Help] [answered]
Hello there,
i wanted to set up authentication with API Guards in v5, but after following the [official tutorial](https://preview.adonisjs.com/guides/auth/setup) i don't get a migration file
for the `api_token` table and also my `config/auth.ts`:

```typescript
const authConfig: AuthConfig = {
  guard: '',
  list: {
  },
}
```
and `contracts/auth.ts`:

```typescript
interface GuardsList {
}
```
are missing configuration.

Also the links in `config/auth.ts` and `contract/auth.ts` are broken. I fixed this manually making use of the [templates](https://github.com/adonisjs/auth/tree/develop/templates) in the adonisjs/auth repo, but it does not seem to be expected behaviour.

**Comment by thetutlage** [comment, marked as answer]
What steps you followed? Seems to be working fine for everyone else

**Reply by christopherfriedrich** [reply]
I first added adonisjs/lucid@alpha and executed the invoke command for it. Then i added adonisjs/auth@alpha and executed its invoke command with the following selections:

![image](https://user-images.githubusercontent.com/35369177/88397656-8eb91600-cdc4-11ea-94e5-5076faa073ca.png)

I tried this multiple times getting the same results.

**Reply by thetutlage** [reply]
The image speaks for itself 

![88397656-8eb91600-cdc4-11ea-94e5-5076faa073ca](https://user-images.githubusercontent.com/1706381/88397924-3f81de00-cde2-11ea-9758-2095ae1b9259.png)

**Reply by christopherfriedrich** [reply]
Found the error silly me 😄

**Reply by thetutlage** [reply]
Make sure you press space to select the authentication guard. The prompt says `select using space`.

<img width="703" alt="Screen Shot 2020-07-24 at 7 11 49 PM" src="https://user-images.githubusercontent.com/1706381/88398001-5b857f80-cde2-11ea-9d43-91938de4e3af.png">
